ReText/tab.py

```diff
@@ -188,3 +189,30 @@ def installFakeVimHandler(self):
 			fakeVimEditor.setQuitAction(self.actionQuit)
 			# TODO: action is bool, really call remove?
 			self.p.actionFakeVimMode.triggered.connect(fakeVimEditor.remove)
+
+
+class ReTextPreview(QTextBrowser):
+	"""
+	When links like [test](test) are clicked, the file test.md is opened.
+	It has to be located next to the current opened file.
+	Relative pathes like [test](../test) or [test](folder/test) are also possible.
+	"""
+
+	def __init__(self, tab):
+		QTextBrowser.__init__(self)
+		self.tab = tab
+		self.setOpenLinks(False)  # if set to True, links to other files will unsuccesfully be opened as anchors
+		self.anchorClicked.connect(self.openInternal)
+
+	def openInternal(self, link):
+		if link.url().startswith('http'):
```

Comment:
How about using `not link.url().isRelative()` instead? It is better because:

* It will fail on different protocols, not only http (i.e. ftp, torrent).
* It will not fail on files named i.e. http.mkd :)